.github/workflows/update_news_zenodo.yml

```diff
@@ -44,12 +44,12 @@ jobs:
           echo "Permission level: $PERMISSION"
           
           # Check if reviewer has sufficient permissions
-          HAS_PERMISSION=false
           if [ "$PERMISSION" == "admin" ] || [ "$PERMISSION" == "maintain" ] || [ "$PERMISSION" == "owner" ]; then
             echo "✅ User has sufficient rights (${PERMISSION})"
-            HAS_PERMISSION=true
           else
             echo "❌ User does not have required permissions"
+            echo "approvals_ok=false" >> $GITHUB_OUTPUT
```

Comment:
Ik ben niet zeker dat die variabele behouden blijft als we de job eindigen met een error.  Maar laten we het zo testen, en als het niet werkt, dan ga je bij de volgende job misschien best ook testen of de variabele bestaat.